relevanceai/dataset_api/dataset_series.py

```diff
@@ -193,7 +193,7 @@ def all(
             show_progress_bar=show_progress_bar,
         )
 
-    def vectorize(self, model):
+    def vectorize(self, model, run_head: bool = False):
```

Comment:
Don't think I follow - what is this for sorry?

Reply:
In https://linear.app/relevance/issue/CUS-170/testing-sdk-core-v0310, @customersuccessteam suggests, "After write operations (vectorise, df.apply , bulk_apply), we should retrieve or update the underlying df obj w/ new cols for a more intuitive UX :)"